packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -198,6 +198,7 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     menuButtonRef: messageActionButtonRef,
     // Force show the action button while the flyout is open (otherwise this will dismiss when the pointer is hovered over the flyout)
     forceShow: chatMessageActionFlyoutTarget === messageActionButtonRef,
+    menuExpanded: chatMessageActionFlyoutTarget === messageActionButtonRef,
```

Comment:
Do we need this; it is exactly the same logic as the forceShow?

Reply:
Removed `forceShow` prop